src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -172,7 +175,16 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
                 if (hubMethodInvocationMessage.ArgumentBindingException != null)
                 {
                     Log.FailedInvokingHubMethod(_logger, hubMethodInvocationMessage.Target, hubMethodInvocationMessage.ArgumentBindingException);
-                    await SendInvocationError(hubMethodInvocationMessage, connection, $"Failed to invoke '{hubMethodInvocationMessage.Target}'. {hubMethodInvocationMessage.ArgumentBindingException.Message}");
+                    var errorMessage = string.Empty;
+                    if (!_hubOptions.EnableDetailedErrors)
+                    {
+                        errorMessage = "some error here";
```

Comment:
I think the pattern of just cutting off the detailed message is sufficient. I do think both messages need a little work though so:

* DetailedErrors off: "Failed to invoke '{Target}' due to an error on the server."
* DetailedErrors on: "Failed to invoke '{Target}' due to an error on the server: {Details}"